instructors/general-lesson-information.Rmd

```diff
@@ -0,0 +1,146 @@
+
+**Example dataset:** <a href="../files/exampleDataObject.csv"> exampleDataObject.csv</a>
+
+<a href="../files/exampleDataObjectColor.png" target="popup" onclick="window.open('../docs/files/exampleDataObjectColor.png,'popup','width=1200'); return false;" onMouseOver="document.exampleData.src='../../docs/files/CLICKexampleDataObjectColor.png';" onMouseOut="document.exampleData.src='../../docs/files/exampleDataObjectColor.png';"><span><img src="files/exampleDataObjectColor.png" alt="Screenshot of example data object." name="exampleData"/></span></a>
+
+<br>
+
+### Learning Objectives
+Our learning objectives follow the model of <a href="https://archive.org/details/taxonomyofeducat0000bloo_o9o7" target="_blank">Bloom <i>et al.</i> (1956)</a> and use the measurable verbs proposed by <a href="https://doi.org/10.3390/educsci6040037" target="_blank">Stanny <i>et al.</i> (2016)</a>.
+
+<br>
+
+### Episode 0: Housekeeping
+In the Instructor guide, we included sections labelled "Episode 0: Housekeeping". These sections are not directly related to the lesson subject, but are important (to us) for the overall teaching experience. Episode 0 sections include activities like learner warm-ups and feedback and can be found in the detailed lesson schedule.
+
+<br>
+
+### Lightning Talk
+We made very good experiences with closing the lesson with a **lightning talk**, that focusses on the application of structured metadata in research, gives an insight into data annotation with metadata in practice, or demonstrates a recent (scientific) software development in the metadata field. For this section, we usually invite an external speaker and assign **45 min** including discussions to the schedule of day 2. 
+
+In our 2 day approach, we end at lunch time on the second day, which allows for extra time for discussions with the speaker. The lightning talk so far got great feedback from the learners.
+
+<br>
+
+### Get in Touch 
+
+You want to contribute to this lesson? Please open an issue in this lesson's Github repository:<br>
+<a href="https://github.com/carpentries-incubator/scientific-metadata" target="_blank">
+  <span>
+    <img src="fig/github.svg" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="#383838" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" aria-hidden="true" style="float:left; padding-right:5px;"/>Github Repository
+  </span>
+</a>
+<br>
+You want to share your experiences or have questions on this lesson? You can get in touch with the creators and maintainers by mail:<br>
+<a href="mailto:HMC@fz-juelich.de?subject=[Carpentries Incubator] Fundamentals of scientific metadata - get in touch"><span><img src="fig/mail.svg" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="#383838" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" aria-hidden="true" style="float:left; padding-right:5px;"/> HMC at fz-juelich.de</span></a>
```

Comment:
why writing out the `at`?  The real email is in the HTML anyway, this does not protect from spambots. 

Either obfuscate properly, or just write `@`